modules/renderer/background.js

```diff
@@ -30,10 +31,82 @@ export function rendererBackground(context) {
 
 
   function ensureImageryIndex() {
-    return fileFetcher.get('imagery')
+    return fileFetcher.get('wayback')
```

Comment:
I don’t feel too great about blocking all the imagery on Esri’s endpoint. It’s a reliable endpoint, but anything can happen in networking. As tail work, we should figure out how to defer this fetch and regenerate the imagery index once it comes in.